webpack/scenes/Subscriptions/components/SubscriptionsTable/SubscriptionsTableSchema.js

```diff
@@ -30,13 +30,13 @@ export const createSubscriptionsTableSchema = (
     property: 'select',
     header: {
       label: __('Select all rows'),
```

Comment:
I don't think we should carry the `hasPermission` variable name into this part of the code since it's not related to permissions at all at this point. Would `selectionEnabled` be a better name?